src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,81 @@
+package duke.tasks;
+
+import duke.main.DukeException;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static java.time.temporal.TemporalAdjusters.next;
+
+public class Deadline extends Task {
+    private LocalDateTime due;
+    private static final DateTimeFormatter FORMAT_LIST[] = {
+            DateTimeFormatter.ofPattern("dd/MM/yyyy"),
+            DateTimeFormatter.ofPattern("dd-MMM-yyyy"),
+            DateTimeFormatter.ofPattern("MM dd, yyyy"),
+            DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm:ss"),
+            DateTimeFormatter.ofPattern("dd-MMM-yyyy HH:mm:ss"),
+            DateTimeFormatter.ofPattern("dd/MM/yy"),
+            DateTimeFormatter.ofPattern("dd-MMM-yy")
+    };
+
+    public Deadline(String s, String deadline) throws DukeException {
+        super(s);
+        tryReadDate(deadline);
+    }
+
+    /**
+     * tryReadDate - Attempts to convert the user supplied date string to a machine date
```

Comment:
For this line of javadoc, you don't have to specify the function name. You can begin with `Attempts to convert..`

Reply:
The same applies to all your other classes as I've noticed this is repeated.